### PR TITLE
Add battery tabular modifications

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx [hunk 1]
```diff
@@ -53,15 +53,19 @@ const TabularModificationForm = () => {
...
-        [intl, watchType]
-    );
+    const explanationComment = useMemo(() => {
+        if (watchType === EQUIPMENT_TYPES.GENERATOR) {
+            return intl.formatMessage({
+                id: 'TabularModificationGeneratorSkeletonComment',
+            });
+        } else if (watchType === EQUIPMENT_TYPES.BATTERY) {
+            return intl.formatMessage({
+                id: 'TabularModificationBatterySkeletonComment',
```
Comment (dbraquart) [inline src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx:63]: I think we dont need to manage a comment in the CSV file if there is nothing in the comment (unlike generator case). For instance there is no comment in the voltage-level case.
We can remove it.

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: I think the CSV comment is not required
tests: ok

File: src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx (hunk 1 above)
Comment (FranckLecuyer) [reply src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx]: Removed

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Pull request merged**